packages/vsx-registry/src/browser/vsx-extension-editor.tsx

```diff
@@ -39,11 +42,16 @@ export class VSXExtensionEditor extends ReactWidget {
         this.updateTitle();
         this.title.iconClass = 'fa fa-puzzle-piece';
         this.node.tabIndex = -1;
-
         this.update();
         this.toDispose.push(this.model.onDidChange(() => this.update()));
     }
 
+    async getScrollContainer(): Promise<HTMLElement> {
+        await waitForRevealed(this);
```

Comment:
looks fragile, one should rather pass deferred promise into the rect component and resolve in ref assignment

Reply:
@akosyakov Thanks for the suggestion, I did not know about deferred technique but went ahead and implemented it according to my understanding